### PR TITLE
CORE-2019: added a changelog

### PR DESCRIPTION
**Pull request by slr71**
This change log is intended to be used to list changes to the Discovery Environment for a specific release.

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+All notable changes to this project will be documented in this file. The format is based on [Keep a Changelog][1].
+
+## [Unreleased]
+
+### Added
+
+- The DE now has the ability to generate HTML source for a badge that can be used to instantly launch an analysis with
+  predefined parameters.
+- When an analysis launch link is used for an instant launch, the DE will now launch the analysis immediately rather
+  than display the analysis launhch wizard.
```
Comment (ianmcorvidae) [inline CHANGELOG.md:12]: `launhch` tpyo

File: CHANGELOG.md [hunk 2]
```diff
@@ -0,0 +1,36 @@
+# CyVerse Discovery Environment (DE) Changelog
+
+All notable changes to this project will be documented in this file. The format is based on [Keep a Changelog][1].
+
+## [Unreleased]
+
+### Added
+
+- The DE now has the ability to generate HTML source for a badge that can be used to instantly launch an analysis with
+  predefined parameters.
```
Comment (ianmcorvidae) [inline CHANGELOG.md:10]: This is still in-progress but pretty close to ready, so it's probably fine to have in here. We've got a month, so.

**Review by ianmcorvidae — COMMENTED** [review]
looks basically fine, you said you were worried about typos so I found one, at least :laughing:

File: CHANGELOG.md [hunk 3]
```diff
@@ -0,0 +1,36 @@
...
+### Changed
+
+- Dependency updates for most microservices.
+- The options to terminate and extend the time limit of a job on the analysis summary page are now buttons if there is
+  sufficient room on the display.
+- The option to request help for an analysis has been moved into the ellipsis menu on the analysis summary page.
+- When launching an analysis, the default setting for maximum CPUs requested will be set to 4 CPUs rather than being set
+  to the highest number of CPUs that can be requested for the app.
+- When an app has multiple versions, anyone with permission to modify the app can change the order of the app versions
+  in order to ensure that the most commonly used app versions stay close to the top of the list.
```
Comment (psarando) [inline CHANGELOG.md:25]: Should this be considered a new feature under `Added`? 🤔

File: CHANGELOG.md (hunk 2 above)
Comment (slr71) [reply CHANGELOG.md:10]: Sounds good. Feel free to let me know if it doesn't look like it's going to be ready for the December release. We can always remove the bullet point if we have to.

**Comment by slr71** [comment]
Thanks for the reviews!

**Pull request merged**